x-pack/metricbeat/module/aws/_meta/config.yml

```diff
@@ -16,3 +16,18 @@
   secret_access_key: '${AWS_SECRET_ACCESS_KEY:""}'
   session_token: '${AWS_SESSION_TOKEN:""}'
   default_region: '${AWS_REGION:us-west-1}'
+- module: aws
+  period: 300s
+  metricsets:
+    - "cloudwatch"
```

Comment:
I see this one now, there is no need to use quotes here, just `- cloudwatch`. This apply to the whole file

Reply:
Done, thanks for pointing it out.